app/styles/components/_pop-in-campagns.scss

```diff
@@ -0,0 +1,156 @@
+.pop-in-campagns {
```

Comment:
Globalement, mettre les font et les couleurs dans des variables, mais aussi séparer + le style et le positionnement.
Il y a trop d'info dans le style de .pop-in-campagns.
```
  bottom: 10px;
  flex-direction: row;
...
```
sont des info de positionnement.

Il faudrait faire 2 classes, une `pop-in-campagns` pour le style visuel et `pop-in-bottom` pour la position.

Cette idée peut marcher pour les autres classes du fichier